Makefile

```diff
@@ -0,0 +1,48 @@
+# No spaces in STACK_NAME
```

Comment:
There has been a significant amount of effort across all of the other teams to standardize on the [ADR-0000](https://github.com/azavea/architecture/blob/master/doc/arch/adr-0000-scripts-to-rule-them-all.md) directory structure. For this project, I think at least the following 3 scripts would be useful for the deployment pipeline:

- https://github.com/azavea/raster-foundry/blob/develop/scripts/cibuild (build container images)
- https://github.com/azavea/raster-foundry/blob/develop/scripts/cipublish (publish container images)
- https://github.com/azavea/raster-foundry-deployment/blob/develop/scripts/infra (plan and apply AWS resource changes)